file_formats/cfg_format.mli

```diff
@@ -25,7 +25,7 @@ type cfg_item_info =
 
 type cfg_unit_info =
   {
-    mutable unit_name : string;
+    mutable unit_name : Compilation_unit.t;
```

Comment:
Same comment as for `Linear_format`.

Reply:
Kept the `Compilation_unit.t` and removed `for_pack`.